### PR TITLE
Bring all Flexsave docs together

### PR DESCRIPTION
**Pull request by norosa**
NOTE: This change is a proposal. I am staging it here so you can interact with it and see how you feel. I expect my reviewers to have comments, suggestions, requests, and so on—all fine.

I aim to present a new way of interacting with the Flexsave docs to see if it sparks any ideas for additional improvements.

---

Previously, the bulk of our Flexsave docs (and the info customers will need, regardless of their user journey) was located in the top-level CMP section. Then, the new onboarding docs were split off to make a new top-level Flexsave section.

This bifurcation left the new Flexsave section looking very bare. And any time we want to reference info covered by the CMP docs, we would have to either:

- Copy it (violating the DRY principle and probably upsetting Google's canonical page algorithm), or

- We would have to sprinkle links everywhere leading back to the CMP docs (which would be disorienting for readers)

In the long term, we plan to combine all of the Flexsave docs and present them as a single product.

I figure now is as good a time as any to make that switch.

Accordingly, I have brought everything together to form a new top-level Flexsave section containing all of our Flexsave docs.

I experimented with the sidebar structure to find one that works for the content we have right now.

I know we agreed on "onboarding," but in this context, I think "sign up" is the operative verb. It makes more sense in the context of the sidebar tree structure. Onboarding is still there but as a component of the sign-up process.

Notes:

- I added a replacement "Flexsave" link to the CMP sidebar to mimic the old section so that CMP users are still aware of it as a feature within the CMP.

  However, selecting that nav item takes you to the new top-level Flexsave section. It is clear from the UI that you have jumped to an entirely new section (and can return if you want to).

- I know we previously agreed on a set of URLs for the engineers to use in the CMP. We can continue to use those URLs with the power of HTTP redirects.

  However, if there is still time, I can generate a new list of URLs for the engineers.

- I added a placeholder page when you select the "Sign up" link from the sidebar. My intention here is to get us thinking bout how we could use this page to communicate who the content is for.

### CODE REVIEW

File: website/docs/flexsave/csp/gcp.mdx [hunk 1]
```diff
@@ -1,19 +1,20 @@
 ---
 sidebar_position: 3
-description: A fundamentally new way to save money on your Google Cloud workloads without making long-term commitments
+sidebar_label: Google Compute Platform
+description: A fundamentally new way to save money on your Google Compute Platform workloads without making long-term commitments
 ---
 
-# Flexsave for GCP
+# Flexsave for Google Compute Platform
```
Comment (azure562) [inline website/docs/flexsave/csp/gcp.mdx:7]: GCP is Google Cloud Platform, which provides cloud computing services

**Comment by azure562** [comment]
About "Sign up", there is a sign-up screen in Figma design https://www.figma.com/proto/ekiUK3fw02NftDcRat4NSa/Flexsave-for-non-resold-customers?page-id=3%3A1587&node-id=205%3A8214&viewport=3123%2C538%2C0.23&scaling=scale-down&starting-point-node-id=205%3A8157

It might be confusing to have "Sign up" in Help Center covering all the other steps

**Comment by norosa** [comment]
@azure562 hmm, I think this strengthens the case for using "Sign up" here. Because in the flows, it seems to me that the flow can be summarised as "Sign up for Flexsave for X", i.e., this is a special sign-up that signs you up for a Flexsave account.

Would you be happy with me leaving this in and requesting a review/discussion with @ryansosin once we have it on the staging site to get a feel for it?

Changing the wording is very easy. It's just a directory rename.

**Comment by norosa** [comment]
@azure562 hmm, I think this strengthens the case for using "Sign up" here. Because in the flows, it seems to me that the flow can be summarised as "Sign up for Flexsave for X", i.e., this is a special sign-up that signs you up for a Flexsave account.

Would you be happy with me leaving this in and requesting a review/discussion with @ryansosin once we have it on the staging site to get a feel for it?

Changing the wording is very easy. It's just a directory rename.

File: website/docs/flexsave/csp/gcp.mdx (hunk 1 above)
Comment (norosa) [reply website/docs/flexsave/csp/gcp.mdx:7]: Good catch, thanks!

**Comment by norosa** [comment]
I will review with @azure562 and @ryansosin later today. More than happy to revert some of the changes if there is a consensus to remove them. Thankfully, reorganizing the sidebar (or renaming things in the sidebar) is trivial, so I believe this PR constitutes a "small reversible step"[^1].

[^1]: "Small reversible steps" is one of [the key tenets of the Apache Way][apache-way] (PDF). I realize this isn't an Apache project, but I've been an Apache contributor for a long time, and they have a lot of good ideas that can be borrowed in other contexts. 

[apache-way]: https://servicecomb.apache.org/assets/slides/20190920/EN/The_Apache_Way_En.pdf

**Pull request merged**